run-dljc.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+# This script runs the Checker Framework on each of a list of projects.
+
+### Usage
+
+# - Move this script to an experiment directory.
+# - Make a file containing a list of git repositories, one per line.
+# - Set variables in the "Configuration" section below.
+# - Run the following command:
+#     bash run-dljc.sh OUTDIR INLIST
+#
+# Where OUTDIR is a name for the experiment you're running, and
+# INLIST is the file containing list of git repositories.
+#
+# This script works as follows:
+# - Checkout each repository into OUTDIR/.
+# - Use do-like-javac to build it once, then
+#   replay the javac commands with checkers enabled.
+# - Place a copy of the results in the directory OUTDIR-results/.
+
+
+### Configuration
+
+# JAVA_HOME must point to a Java 8 JDK.
+if [ "x${JAVA_HOME}" = "x" ]; then
+    JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk
+fi
+
+# What typecheckers should be run?
+CHECKERS=org.checkerframework.checker.builder.TypesafeBuilderChecker
+
+# If you want to use a custom typechecker or typecheckers,
+# define their dependencies here.
+CHECKER_LIB=/homes/gws/kelloggm/image-sniping-oss/typesafe-builder-checker/build/libs/typesafe-builder-checker.jar:/homes/gws/kelloggm/.m2/repository/org/springframework/spring-expression/5.1.7.RELEASE/spring-expression-5.1.7.RELEASE.jar:/homes/gws/kelloggm/.m2/repository/org/springframework/spring-core/5.1.7.RELEASE/spring-core-5.1.7.RELEASE.jar:/homes/gws/kelloggm/.m2/repository/org/springframework/spring-jcl/5.1.7.RELEASE/spring-jcl-5.1.7.RELEASE.jar:
+
+# Should any stub files be used?
+STUBS=/homes/gws/kelloggm/image-sniping-oss/typesafe-builder-checker/stubs
+
+
+### Script
+
+# clone DLJC if it's not present
+if [ ! -d do-like-javac ]; then
+    git clone https://github.com/kelloggm/do-like-javac
+fi
+
+PWD=`pwd`
+
+DLJC=${PWD}/do-like-javac/dljc
+
+if [ "x${CHECKERFRAMEWORK}" = "x" ]; then
+
+    if [ ! -d checker-framework ]; then
+	git clone https://github.com/typetools/checker-framework
```

Comment:
Does this work?  Wouldn't it need to be built?  I would just make one of the prerequisites be a $CHECKERFRAMEWORK environment variable.